pages/_app.jsx

```diff
@@ -1,7 +1,50 @@
+import { useEffect, useState } from "react";
 import "@/styles/globals.scss";
 import Header from "@/components/Header";
 
 export default function App({ Component, pageProps }) {
+	const [theme, setTheme] = useState("bright");
+	const [fontSize, setFontSize] = useState(16);
+
+	const brightPrimary = "#EC7373";
```

Comment:
these variables don't belong to this file. mentioned this before but you should separate them to a different file or colocate them inside the themeContext

Reply:
noted